le.go

```diff
@@ -138,9 +138,21 @@ func (logger *Logger) Flags() int {
 
 // Output does the actual writing to the TCP connection
 func (logger *Logger) Output(calldepth int, s string) error {
```

Comment:
I'm thinking the reconnect logic would make more sense in Write() where it already does a connection test.